apps/mobile-alphabet/src/app/App.spec.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+
+import App from './App';
+
+test('renders correctly', () => {
+    const { getByTestId } = render(<App />);
+    expect(getByTestId('heading')).toHaveTextContent('Welcome');
```

Comment:
Obviously, this is boilerplate. But you might update it once you have decided on a welcome message for your entry point.